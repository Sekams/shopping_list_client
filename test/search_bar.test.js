import React from 'react';
import SearchBar from '../src/components/search_bar'

test("renders the search_bar component", () => {
    const wrapper = shallow(
        <SearchBar />
    );
    expect(wrapper).toMatchSnapshot();
});