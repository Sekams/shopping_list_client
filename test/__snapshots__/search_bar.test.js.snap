// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the search_bar component 1`] = `
<div
  className="search-bar"
>
  <input
    className="form-control"
    onChange={[Function]}
    placeholder="Search"
    value=""
  />
</div>
`;
